### PR TITLE
fix(STONEINTG-574) fix log message about number of integration tests

### PR DESCRIPTION
**Pull request by sonam1412**
## Maintainers will complete the following section

- [x] Commit messages are descriptive enough ([hints](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/))
- [x] Code coverage from testing does not decrease and new code is covered
- [x] [Controllers diagrams](https://github.com/redhat-appstudio/integration-service/tree/main/docs) are updated when PR changes controllers code  (if applicable)

### CODE REVIEW

File: controllers/snapshot/snapshot_adapter.go [hunk 1]
```diff
@@ -82,7 +82,9 @@ func (a *Adapter) EnsureAllIntegrationTestPipelinesExist() (controller.Operation
 	}
 
 	if integrationTestScenarios != nil {
-		a.logger.Info("Found IntegrationTestScenarios for application",
+		var numberOfIntegrationTestScenario = len(*integrationTestScenarios)
+		a.logger.Info(
+			fmt.Sprintf("Found %v IntegrationTestScenarios for application", numberOfIntegrationTestScenario),
```
Comment (MartinBasti) [inline controllers/snapshot/snapshot_adapter.go:87]: Can be simplified, also `%d` should be used for numbers
```suggestion
		a.logger.Info(fmt.Sprintf("Found %d IntegrationTestScenarios for application", len(numberOfIntegrationTestScenario)),
```

**Review by hongweiliu17 — APPROVED** [review]
LGTM

**Comment by sonam1412** [comment]
tested on local cluster, see  results [here](https://issues.redhat.com/browse/STONEINTG-574?focusedId=23030443&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-23030443)

**Review by MartinBasti — APPROVED** [review]

**Pull request merged**